quiz-app/css/question.css

```diff
@@ -0,0 +1,150 @@
+.question-container{
+    display: flex;
+    align-items: center;
+    justify-content:center;
+    height: 100vh;
+}
+ .quiz-box{
+    width: 550px;
+    background: var(--COLOR-WHITE);
+    border-radius: 0.5rem;
+}
+
+.quiz-box header{
+    height: 70px;
+    padding: 0 30px;
+    background: var(--COLOR-WHITE);
+    border-radius: 5px 5px 0 0;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
```

Comment:
same use flex as helper class and overwrite specific changes in the class of element you are applying the properties